### PR TITLE
feat: show active events in user menu

### PR DESCRIPTION
**Pull request by snitin315**
Resolve https://github.com/opencollective/opencollective/issues/3416

# Description
Show active events if any.

# Screenshots
Screenshot at 2020-08-23 08-26-38

### CODE REVIEW

**Review by Betree — COMMENTED** [review]
It's good! Two possible enhancements:
- Events should be displayed after `Collectives`, conceptually Events are "children" of collectives
- The proper way to check if an event is over is with the `isPastEvent` helper (tho it's good to check `isActive` too).

**Comment by snitin315** [comment]
@Betree I have updated the PR as suggested.

Screenshot at 2020-08-24 13-08-34

File: components/TopBarProfileMenu.js [hunk 1]
```diff
@@ -329,6 +336,28 @@ class TopBarProfileMenu extends React.Component {
                 </P>
               </Box>
             )}
+            {events.length > 0 && (
+              <div>
+                <Flex alignItems="center">
```
Comment (Betree) [inline components/TopBarProfileMenu.js:341]: To match the other margins:

```suggestion
                <Flex alignItems="center" mt={3}>
```

**Review by Betree — COMMENTED** [review]
Looks good!

**Review by Betree — COMMENTED** [review]
We also need to add `endsAt` for collectives in `loggedInUserQuery` in `lib/graphql/queries.js`, otherwise we'll still display past events.

**Pull request merged**